framework/filters/PageCache.php

```diff
@@ -123,6 +124,12 @@ class PageCache extends ActionFilter
      * @since 2.0.4
      */
     public $cacheHeaders = true;
+    /**
+     * @var array a list of placeholders for embedding dynamic contents. This property
+     * is used internally to implement the content caching feature. Do not modify it.
+     * @since 2.0.10
+     */
+    public $dynamicPlaceholders;
```

Comment:
Not sure… I guess it would require changes in 'View' and any other classes implementing a dynamic content.